recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
-{% set version = "3.3.9" %}
+{% set version = "3.5.0" %}
+{% set sha256 = "beb91419245395bd69a4a6edad5ca3ec1a8b64e41457672dc687c173a495f034" %}
```

Comment:
I prefer to only use variables for stuff that occurs more than once in the recipe.